### ISSUE TITLE
error: clGetDeviceIDs

### ISSUE THREAD

**Issue by brizio1971** [Genoil/ZECMiner#26]
I get thi error running ver 0.4 on windows 10 and RX470 card

**Comment by Genoil** [comment]
Did you try adding -P 1?

**Comment by juanjpl88** [comment]
hi

I have exactly the same error. I am trying to run a rig of 5xmsi r9390 with h97 anniversary

**Comment by dleduc86** [comment]
I am getting that error 5 cards AMD rx480....no idea why....i tried the -p1

**Comment by 1amgoingin** [comment]
I am also getting this error on AMD HD 6670 -P 1 doesn't help

**Comment by Bpar** [comment]
use -P 1 or -P 0 and have the same result :(

**Comment by du44** [comment]
Have this trouble too with Sapphire rx 480 8GB ;( -P 0 or -P 1 not solved this problem

**Comment by lavrfedotov** [comment]
i have same problen with clGetDeviceIDs
but after -P 1  mine starts and then crash on second job for no reason
![image](https://cloud.githubusercontent.com/assets/23178121/19940279/2d7d7564-a13d-11e6-888d-890f1070649f.png)

**Comment by lavrfedotov** [comment]
problem solved

**Comment by lavrfedotov** [comment]
for all who has this error
here is my bat
`genoil.exe -c us1-zcash.flypool.org:3333 -u t1QeYajjd9N2YgW9PjGVpynpq3FQAMMVtEq.lavr23 -p z -P 1 -g 0 1`
for 2 cards 470 saph

**Comment by Bpar** [comment]
Yes it work if you have few gpu devices. But if you have only integrated card it's doesn't work :(

**Comment by du44** [comment]
didnt work at system with one gtx 1070 and one rx 480 8GB(Sapphire) - bat only for amd,

**Comment by 1amgoingin** [comment]
Still not fixed in v0.6 come on guys it can't be that hard can it?
